schemas/JSON/manifests/v1.2.0/manifest.installer.1.2.0.json

```diff
@@ -88,6 +88,42 @@
       ],
       "description": "Scope indicates if the installer is per user or per machine"
     },
+    "NestedInstallerType": {
+      "type": [ "string", "null" ],
+      "enum": [
+        "msix",
+        "msi",
+        "appx",
+        "exe",
+        "inno",
+        "nullsoft",
+        "wix",
+        "burn",
+        "pwa",
+        "portable"
+      ],
+      "description": "Enumeration of supported nested installer types."
+    },
+    "NestedInstallerFile": {
+      "type": "object",
+      "properties": {
+        "RelativeFilePath": {
+          "type": [ "string" ],
+          "minLength": 1,
+          "maxLength": 260,
+          "description": "The relative path to a nested installer file contained inside an archive."
+        },
+        "PortableCommandAlias": {
+          "type": [ "string", "null" ],
+          "minLength": 1,
+          "maxLength": 40,
+          "description":  "The command alias to to be used for calling the package. Only applies to a portable package."
+        }
+      },
+      "maxItems": 16,

```

Comment:
Seems low; is there a reason to limit this?

Reply:
Removed the limit